sdks/wasm/xunit-exclusions.rsp

```diff
@@ -9,7 +9,7 @@
 -noclass System.Tests.ProgressTests
 -nonamespace System.Threading.Tests
 -nonamespace System.Threading.Tasks
--nonamespace Sytem.Threading.Tasks
+-nonamespace System.Threading.Tasks
```

Comment:
Thats a typo in the test class name, not here.

Reply:
Added a comment

Reply:
Removed the comment because its can be passed on the xunit command line. The original issue with name can be fixed in related corefx sources.

Reply:
Filed in https://github.com/dotnet/corefx/issues/40973